sidecar_acceptor_test.go

```diff
@@ -32,136 +31,215 @@ var (
 	}
 )
 
-func newTestDB(t *testing.T) (*clientdb.DB, func()) {
-	tempDir, err := ioutil.TempDir("", "client-db")
-	require.NoError(t, err)
-
-	db, err := clientdb.New(tempDir, clientdb.DBFilename)
-	if err != nil {
-		require.NoError(t, os.RemoveAll(tempDir))
-		t.Fatalf("unable to create new db: %v", err)
-	}
+var registerSidecarEmptySetter = func(
```

Comment:
double nit: could just be an empty func, no need to declare it as a `var`. Also, the first argument should be on the same line if it fits, otherwise this looks like a function _call_ instead of a _definition_ at first glance.